src/function.py

```diff
@@ -453,6 +461,11 @@ def _package_log_payload(data):
         }
     ]
 
+    nr_tags_str = _get_newrelic_tags()
+    if nr_tags_str:
+        nr_tags = dict(item.split(":") for item in _get_newrelic_tags().split(";"))
```

Comment:
Let's also filter out collisions; the user should not overwrite the `plugin` or `aws` keys.

Reply:
I have made this change as well.